{
  logger.error("RetryOperationCallback call was interrupted for account {}",paymentStateContext.getAccount().getExternalKey());
  return new OperationException(e,getOperationResultOnException(paymentStateContext));
}
